SharpNoise.Tests/ModuleTests/DiscreteTests.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SharpNoise.Modules;
+
+namespace SharpNoise.Tests.ModuleTests
+{
+    [TestClass]
+    public class DiscreteTests
+    {
+        Random random;
+
+        const Byte VALUES = 4;
+
+        public void DiscreteMainTestCode(Module source, Byte Count = VALUES)

```

Comment:
The `Count` argument has the same name as the count variable in the for loop below, only it is capitalized. Please make that a bit less ambiguous.

Also why is the argument a Byte instead of a uint16 which would seem more appropriate since Discrete.Values is also a uint16.